i18n/en-US.properties

```diff
@@ -34,12 +34,6 @@ be.activitySidebarFilter.status.tasks = Tasks
 be.add = Add
 # Text to display when app is disabled by applied access policy
 be.additionalTab.blockedByShieldAccessPolicy = Use of this app is blocked due to a security policy.
-# Error message when an annotation deletion fails
```

Comment:
Do we know why this file was updated?

Reply:
It always seems to be updated whenever I run `yarn build`. I can leave it out if you'd like, since I don't have a good idea of when we include changes from build vs not. When do we want to include changes vs not?

Reply:
I think these types of changes should be included when we delete the reference in the code. You can still see the translation used in the code here https://github.com/box/box-ui-elements/blob/d40456caa2d91179c89125f4f045f74632cb5f5a/src/elements/content-sidebar/activity-feed/annotation-thread/errors.js#L16. It's probably deleted when you do yarn:build because it's in an object 🤷‍♀️ We should probably change it so that it's not deleted when you run yarn:build but in a different PR.